doc/deoplete.txt

```diff
@@ -429,6 +429,8 @@ member						*deoplete-source-member*
 tag						*deoplete-source-tag*
 		It collects keywords from |ctags| files.
 		Note: It only supports UTF-8 encoding tag file.
+		Note: https://github.com/Shougo/neoinclude.vim is required for
+		this.
```

Comment:
The change is wrong.
neoinclude is not required!
If you want to complete the candidates from included files, neoinclude is required.